src/pages/ContactUs.jsx

```diff
@@ -1,12 +1,91 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import ContentContainer from "../components/ContentContainer";
+import InputField from "../components/InputField";
+import Button from "../components/Button";
+import Alert from "../components/Alert";
+import emailjs from "emailjs-com";
+import { EnvelopeIcon } from "@heroicons/react/24/outline";
 
 function ContactUs() {
+  const [alertMessage, setAlertMessage] = useState(null);
+  const [alertType, setAlertType] = useState(null);
+  const [isSubmitting, setIsSubmitting] = useState(false);
+
+  useEffect(() => {
+    if (alertMessage) {
+      const timer = setTimeout(() => {
+        setAlertMessage(null);
+      }, 10000);
+      return () => clearTimeout(timer);
+    }
+  }, [alertMessage]);
+
+  function sendEmail(e) {
+    e.preventDefault();
+    setIsSubmitting(true);
+
+    emailjs.sendForm(
+      'service_5fjjqxm',   // service ID
+      'template_poq4yrh',  // template ID
+      e.target,
+      'oVlgYbBowoOyaum0h'  // public KEY
```

Comment:
Did you confirm this works as intended? A quick look at the documentation seems to require the public key to be passed as an options object instead of as a direct param for `sendForm`

Reply:
Yes! I tested it, and the emails are successfully delivered to the specified email address